### PR TITLE
Tab Staggered Permissions Fixes (Don't merge this PR - Duplicate)

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Tested Running Sample
----------------------------
image

### CODE REVIEW

File: samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-end.cshtml.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public authEnd(IConfiguration configuration)
 	/// </summary>
         public void OnGet()
         {
-            ClientId = _configuration["MicrosoftAppId"];
+            ClientId = _configuration["AzureAd:ClientId"];
```
Comment (Pawank-MSFT) [inline samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-end.cshtml.cs:31]: appsetting.json have different name, "MicrosoftAppId" Please correct it.

File: samples/tab-staggered-permission/csharp/StaggeredPermission/helper/AuthHelper.cs [hunk 2]
```diff
@@ -83,8 +83,8 @@ public static IEnumerable<string> GetValidIssuers(IConfiguration configuration)
         public static async Task<string> GetAccessTokenOnBehalfUserAsync(IConfiguration configuration, IHttpClientFactory httpClientFactory, IHttpContextAccessor httpContextAccessor, string idToken)
         {
             var tenantId = configuration["AzureAd:TenantId"];
-            IConfidentialClientApplication app = ConfidentialClientApplicationBuilder.Create(configuration["AzureAd:MicrosoftAppId"])
-                                                .WithClientSecret(configuration["AzureAd:MicrosoftAppPassword"])
+            IConfidentialClientApplication app = ConfidentialClientApplicationBuilder.Create(configuration["AzureAd:ClientId"])
+                                                .WithClientSecret(configuration["AzureAd:AppSecret"])
```
Comment (Pawank-MSFT) [inline samples/tab-staggered-permission/csharp/StaggeredPermission/helper/AuthHelper.cs:87]: check appsettings.json not found  "AppSecret" and "ClientId"

**Review by Pawank-MSFT — COMMENTED** [review]
please fix comments

File: samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-end.cshtml.cs (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-end.cshtml.cs:31]: fixed

File: samples/tab-staggered-permission/csharp/StaggeredPermission/helper/AuthHelper.cs (hunk 2 above)
Comment (Mohammed-MSFT) [reply samples/tab-staggered-permission/csharp/StaggeredPermission/helper/AuthHelper.cs:87]: fixed

**Comment by Mohammed-MSFT** [comment]
Comments fixed and merge conflicts resolved.

**Pull request closed without merging**